src/components/CitizenData.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Paymethods from "./Paymethods";
+import { BPDCitizen } from "../generated/definitions/BPDCitizen";
+import { format, parseISO } from "date-fns";
+import { useTranslation } from 'react-i18next';
+
+interface CitizenDataProps {
+  resultData: BPDCitizen
+}
+
+function CitizenData(props: CitizenDataProps) {
```

Comment:
ditto. if you don't use React.FC / React.FunctionalComponent you cannot acess children and other core properties